pkg/cache/configmaps.go

```diff
@@ -57,6 +60,24 @@ func GetConfigMap(key interface{}) *corev1.ConfigMap {
 	return nil
 }
 
+// SetConfigMap based on the shared-data-key, which contains the resource's namespace and name, this
+// method can fetch and store it on cache.
+func SetConfigMap(kubeClient kubernetes.Interface, sharedDataKey string) error {
+	ns, name, err := SplitKey(sharedDataKey)
+	if err != nil {
+		return err
+	}
+
+	ctx := context.TODO()
+	cm, err := kubeClient.CoreV1().ConfigMaps(ns).Get(ctx, name, metav1.GetOptions{})
```

Comment:
If you aren't going to use the context anywhere else, why not pass the `context.TODO()` directly?
`cm, err := kubeClient.CoreV1().ConfigMaps(ns).Get(context.TODO(), name, metav1.GetOptions{})`

Reply:
Indeed, please consider latest changes.